flocker/node/script.py

```diff
@@ -301,13 +223,10 @@ class ServeScript(object):
     a Flocker cluster.
     """
     def main(self, reactor, options, volume_service):
-        api_service = create_api_service(
-            TCP4ServerEndpoint(reactor, options["port"]))
-        parent_service = _ServeService(volume_service, api_service)
-        return _main_for_service(reactor, parent_service)
+        return main_for_service(reactor, volume_service)
 
 
-def flocker_serve_main():
+def flocker_volume_main():
```

Comment:
Yeah... just leave it as is, nothing's use it yet anyway, and we can rename it later.